docs-web/src/main/java/com/sismics/docs/rest/resource/ReviewsResource.java

```diff
@@ -0,0 +1,33 @@
+package com.sismics.docs.rest.resource;
+
+import javax.ws.rs.Path;
+
+/**
+ * Reviews REST resources.
+ * 
+ * @author Ziqi Ding
+ */
+@Path("/reviews")
+public class ReviewsResource {
+    /**
+     * Returns the reviews for a document.
+     *
+     * @api {get} /reviews/:id Get a review
+     * @apiName GetReview
+     * @apiGroup Review
+     * @apiParam {String} id Document ID
+     * @apiParam {Booleans} files If true includes files information
+     * @apiSuccess {String} id ID
+     * @apiSuccess {String} title Title
+     * @apiSuccess {String} description Description
+     * @apiSuccess {Object[]} categories Categories
+     * @apiSuccess {Object[]} ratings Ratings
```

Comment:
Could you please be more precise in the description of what’s inside the `categories`/`ratings` arrays? Currently, I don’t know what format I should expect to receive from this API.

Also, it seems to me that this format couldn’t support these edge cases:
* Having multiple workflows with review steps on the same document
* Having comments in the reviews